### PR TITLE
feat: submitOnEnter for numeric keypad Enter key (keyCode = 13)

### PR DESCRIPTION
**Pull request by hnhegde**
feat: submitOnEnter for numeric keypad Enter key

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -42,7 +42,8 @@ export default class EdiText extends Component {
   }
 
   _handleEnter = e => {
-    if (e.keyCode === 'Enter' || e.code === 'Enter' || e.keyCode === 27) {
+    if (e.keyCode === 'Enter' || e.code === 'Enter' || e.keyCode === 27 || \
```
Comment (alioguzhan) [inline src/index.js]: Hey, there is a syntax error here.

Comment (alioguzhan) [reply src/index.js]: Thanks for the PR, btw.

Comment (hnhegde) [reply src/index.js]: Hey @alioguzhan , 
Extremely useful component! Thanks for putting it out! I have made a small attempt to handle an edge case.

Comment (alioguzhan) [reply src/index.js]: Thanks for your contribution! I am glad if you like it. Merging it...

**Comment by alioguzhan** [comment]
@all-contributors please add @hnhegde  for bug and code

**Pull request merged**